variables.tf

```diff
@@ -50,22 +50,46 @@ variable "target_network" {
   default     = ""
 }
 
+variable "description" {
+  description = "domain description ( shown in console )"
+  default     = "domain managed by Terraform"
+  type        = string
+}
+
 variable "type" {
   description = "Type of zone to create, valid values are 'public', 'private', 'forwarding', 'peering'."
   default     = "private"
   type        = string
 }
 
+variable "dnssec_config" {
+  description = "A dnssec_config, please see https://www.terraform.io/docs/providers/google/r/dns_managed_zone.html#dnssec_config"
+  type        = any
+  default     = {}
+}
+
+variable "default_key_specs_key" {
```

Comment:
Can you add a description to this and the following variables?

Reply:
I just added some explanation at least on the object structure, but I'm not sure how to explain clearly without pointinf out to RFCs for people not knowing at all dnssec configuration ... 
But if you think it can be improved, feel free:)